control/Controller.php

```diff
@@ -119,7 +119,6 @@ public function handleRequest(SS_HTTPRequest $request, DataModel $model) {
 		$this->pushCurrent();
 		$this->urlParams = $request->allParams();
 		$this->setRequest($request);
-		$this->response = new SS_HTTPResponse();
```

Comment:
OK - so this is a potential regression here as people are likely to require this property to be populated as part of `onBeforeInit`, `init` and `onAfterInit`